### PR TITLE
Issue #1554

### PR DESCRIPTION
**Pull request by j0rt3g4**
The added suggestion of the script into the PR

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: OneDrive/pre-provision-accounts.md [hunk 1]
```diff
@@ -77,6 +77,37 @@ To verify that OneDrive has been created for your users, see [Get a list of all
    >[!NOTE]
    >If you are pre-provisioning OneDrive for many users, it might take up to 24 hours for the OneDrive locations to be created. If a    user's OneDrive isn't ready after 24 hours, please contact Support.
   
+# Pre-provisioning many users at one time.
```
Comment (JohanFreelancer9) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
# Pre-provisioning many users at the same time
```

File: OneDrive/pre-provision-accounts.md [hunk 2]
```diff
@@ -77,6 +77,37 @@ To verify that OneDrive has been created for your users, see [Get a list of all
...
+
+The following sniped of code will preprovision Onedrive for a lot of users for bigger companies.
```
Comment (JohanFreelancer9) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
The following snipped of code will pre-provision OneDrive for a number of users in bigger companies.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by jordonthreadgill** [comment]
@JohanFreelancer9 Line 107 needs to be updated to $j instead of $ii.

**Comment by j0rt3g4** [comment]
@jordonthreadgill  correction taken and also yours @JohanFreelancer9 .

**Pull request merged**